app/screens/auth/set_up_screen.tsx

```diff
@@ -52,12 +63,18 @@ export default function SetUsername({ navigation }: any) {
         <ThemedView style={styles.smallContainer} colorType="transparent">
           {/* Profile picture */}
 
-          <ThemedIconButton
-            name="person-circle-outline"
-            size={300}
-            colorType="textPrimary"
-            onPress={() => navigation.navigate("Camera")}
-          />
+          <TouchableOpacity onPress={pickImage} style={styles.smallContainer}>
```

Comment:
For the components, it would be good to include the test IDs now for future testing.

Reply:
Absolutely !